### PR TITLE
fix: block events for missing courses

### PR DESCRIPTION
**Pull request by Ian2012**
**Description:**  A setting to allow to send missing courses

**JIRA:** Link to JIRA ticket

**Dependencies:** dependencies on other outstanding PRs, issues, etc.

**Merge deadline:** List merge deadline (if any)

**Installation instructions:** List any non-trivial installation
instructions.

**Testing instructions:**

1. Open page A
2. Do thing B
3. Expect C to happen
4. If D happened instead - check failed.

**Merge checklist:**
- [ ] All reviewers approved
- [ ] CI build is green
- [ ] Version bumped
- [ ] Changelog record added
- [ ] Documentation updated (not only docstrings)
- [ ] Commits are squashed

**Post merge:**
- [ ] Create a tag
- [ ] Check new version is pushed to PyPI after tag-triggered build is
      finished.

**Author concerns:** List any concerns about this PR - inelegant
solutions, hacks, quick-and-dirty implementations, concerns about
migrations, etc.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @Ian2012!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

File: event_routing_backends/helpers.py [hunk 1]
```diff
@@ -149,9 +149,8 @@ def get_course_from_id(course_id):
     course_overviews = get_course_overviews([course_key])
     if course_overviews:
         return course_overviews[0]
-    return {
-        "display_name": "Unknown Course",
-    }
+    raise ValueError(f"Course with id {course_id} does not exist.")
```
Comment (bmtcril) [inline event_routing_backends/helpers.py:152]: Can't remember, will this be caught upstream so it doesn't break bulk export?

Comment (Ian2012) [reply event_routing_backends/helpers.py:152]: Not sure

Comment (Ian2012) [reply event_routing_backends/helpers.py:152]: The only helper that raises an exception is `get_anonymous_user_id` and it raises a `ValueError` exception. It shouldn't break the workflow.

@ziafazal Am I right?

Comment (ziafazal) [reply event_routing_backends/helpers.py:152]: @Ian2012 yes `ValueError` is caught [here](https://github.com/openedx/event-routing-backends/blob/99ede4ede3f71a8769b122b9ec8db50766b8a71d/event_routing_backends/backends/events_router.py#L83) so it should not break.

**Pull request merged**